<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Введите номер договора</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ContractNumberLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Дата начала действия договора:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Date1Line">
         <property name="placeholderText">
          <string>YYYY-MM-DD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="Calendar1Button">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Окончания:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Date2Line">
         <property name="placeholderText">
          <string>YYYY-MM-DD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="Calendar2Button">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Наименование покупателя:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ClientNameLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Наименование перевозчика:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DeliveryAgentNameLine"/>
       </item>
       <item>
        <widget class="QComboBox" name="DeliveryAgentBinaryIndicatorComboBox">
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Выбор товаров</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="2">
           <widget class="QTableWidget" name="AddedProductWithQuantityTableWidget">
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <column>
             <property name="text">
              <string>Наименование</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Стоимость</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Количество</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="AddToChosenProductButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="ProductNameToFilterLine">
            <property name="placeholderText">
             <string>Product name...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="ProductNameToFilterButton">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QTableWidget" name="AllProductToAddTableWidget">
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <column>
             <property name="text">
              <string>Наименование</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Стоимость</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Единицы измерения</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="QuantityOfProductToAddLine">
            <property name="placeholderText">
             <string>Quantity...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="DeleteProductFromAddedButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="UploadToDBButton">
         <property name="text">
          <string>Upload to DB</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
